src/SciAssist/configs/datamodule/dataset_extraction.yaml

```diff
@@ -0,0 +1,9 @@
+_target_: SciAssist.datamodules.dataset_extraction_datamodule.DatasetExtractionModule
+
+data_repo: "data-full"
```

Comment:
Could this 'data-full' be changed to a meaningful name?

Reply:
I set data_repo=None when using local data. Or you may simply delete this parameter. I recommend to use `data_cache_dir` to specify local datasets instead in case of misunderstanding. If you are loading data from a single file, you may use 'data_file'.

Reply:
I guess this issue is not fixed?